README.md

```diff
@@ -72,10 +72,16 @@ All these shader functions are based on publicly available shader functions (ope
   * `linear_scene_depth_*`
   * `distance_fade`
   * `proximity_fade_*`
+  * `world_position_from_depth`
```

Comment:
since we can't know which rendering backend the user will use for developing, we have to supply a `world_position_from_depth_vulkan` as well as a `_gles` variant function. See here https://github.com/godotengine/godot/pull/65160